redbot/cogs/downloader/downloader.py

```diff
@@ -1207,3 +1237,30 @@ def cog_name_from_instance(self, instance: object) -> str:
             msg = _("This command is not provided by a cog.")
 
         await ctx.send(box(msg))
+
+    @staticmethod
+    def format_failed_repos(failed: Iterable[str]) -> str:
+        """Format iterable of ``Repo.name`` into failed message.
+
+        Parameters
+        ----------
+        failed : Iterable
+            Iterable of ``Repo.name``
+
+        Returns
+        -------
+        str
+            formatted message
+        """
+
+        message = (
+            _("Failed to update following repositories:")
+            if len(failed) > 1
+            else _("Failed to update following repository:")
+        )
+        message += " " + humanize_list(tuple(map(inline, failed))) + "\n"
+        message += _(
+            "Repository's branch might have been removed or"
+            " it's no longer accessible at set url. See logs for more information."
```

Comment:
Should be reworded to "**The** repository's branch might have been removed or **the repository is** no longer accessible at **the** set url."